src/Akka.Persistence.Sql.Hosting/Akka.Persistence.Sql.Hosting.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>$(NetStandardLibVersion)</TargetFramework>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Akka.Persistence.Hosting" />
```

Comment:
LGTM, version range for the MSFT.EXT packages is controlled by the base Akka.Hosting library.